kytos/core/controller.py

```diff
@@ -169,6 +170,18 @@ def register_rest_endpoint(self, *options, **kwargs):
         """Method used to return the endpoints registered by APIServer."""
         self.api_server.register_rest_endpoint(*options, **kwargs)
 
+    def restart(self, graceful=True):
+        """Restart Kytos SDN Platform.
+
+        Args:
+            graceful(bool): Represents the way that Kytos will restart.
+        """
+        if self.started_at is not None:
+            self.stop(graceful)
```

Comment:
The `stop()` may take a while (since there are lot's of threads to be stopped). I think that we may need to use a `callback` behavior here, waiting the stop to finish it's procedures and just then starting the controller again.

@beraldoleal any ideas on this?